README.md

```diff
@@ -0,0 +1,140 @@
+but skuld also purges Swift, Cinder and Blueprint resources.
+
+### Deploying one testbed
+By using a fiware-testbed-deploy script (*launch_vm.py*), it is possible to deploy an entire Openstack testbed.
+The script uses a floating IP and creates both a keypair (it saves the SSH private key as
+~/.ssh/testbedskuld_key) and a security group. The security group opens
+the port 22 (SSH) and allows the ICMP traffic. It is also possible to connect
+to any port from a VM using the same security group.
+
+The names of the keypair, the security group and other parameters as the preferable
+floating IP are configurable in the *settings.py* file, but most of the users may
+ignore this file securely. However, to use the script in a different platform
+than FIWARE Lab, probably is necessary to change the parameter about the
+public shared network name.
+
+Although it is already possible to connect to the ubuntu account of the server, the installation is still running
+inside the VM and needs a few minutes to complete. Usually the installation
+process lasts between 10 and 20 minutes. The job is finished after the file
+*config_vars* is copied into the */home/ubuntu* folder inside the virtual machine.
+
+When the installation is finished, the credential may be loaded with *. ~/config_vars*.
+The command *nova list* shows a testing VM that has been created during the installation
+inside the testbed (that is, it is a virtual machine running inside the testbed
+virtual machine). The floating IP 192.168.58.201 is associated to this
+VM (that is the second IP of the pool, the first was assigned to the router). It is
+possible to connect to the server following this steps:
+
+    eval $(ssh-agent -s)
+    ssh-add ~/.ssh/testkey
+    ip a add dev br-ex 192.168.58.1/24
+    ssh cirros@192.168.58.201
+
+Of course, if a new server is instantiated using the same network, there will
+be network visibility between both servers, according the rules of the
+security groups.
+
+     . ~/config_vars
+     NETID =
+     nova boot testvm2 --poll --flavor micro2 --image cirros --nic net-id=$NETID\
+      --key-name testkey --security-groups ssh
+     ssh -A cirros@192.168.58.201
+     $ ssh cirros@192.168.58.3
+
+The micro flavor provides 64MB of RAM, 1 VCPU and 1GB of disk. The micro2 flavor is the
+same, but with 0GB of disk (i.e. a minimal disk to boot the image is created
+but with barely free space)
+
+### Deploying three testbeds
+
+The *deploythreeglances.py* script deploys 3 OpenStack Image Repositories (Glance) which share the same
+Identity Service called Keystone. Each glance is deployed on a different server accessible by a floating IP.
+It is possible to access by SSH Key and its configuration variables are in the file config_vars in
+/home/ubuntu folder inside the server, in the same way that it was explained in "Deploying one testbed"
+section. These 3 Glance testbeds are used for GlanceSync acceptance tests.
```

Comment:
Please put a link to the GlanceSync component in github (README) in order to give visibility of what is it